tables/courses.rb

```diff
@@ -266,8 +266,8 @@ def migrate_settings(source, destination)
     # Migrate pagination setting
     ###################################
     set_unless_default.call([:course_announcements_component, :pagination], :paginate_announcements_count, '50')
-    set_unless_default.call([:course_discussion_topics_component, :pagination], :paginate_comments_count, '50')
-    set_unless_default.call([:course_forums_component, :pagination], :paginate_forums_count, '20')
+    set_unless_default.call([:course_discussion_topics_component, :pagination], :paginate_comments_count, '10')
```

Comment:
Would suggest leaving the pagination count as the original default, thus users will get better performance when viewing those pages.